compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5826,6 +5826,78 @@ TR::Register *OMR::Power::TreeEvaluator::retrieveTOCRegister(TR::Node *node, TR:
    return cg->machine()->getRealRegister(TR::RealRegister::gr2);
 }
 
+TR::Register *OMR::Power::TreeEvaluator::sbyteswapEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT(node->getNumChildren() == 1, "Wrong number of children in sbyteswapEvaluator");
```

Comment:
Is this needed since IL validators will catch errors here?

Reply:
Hmm, I definitely agree that these aren't needed. Looks like all architectures have these checks in their byteswap evaluators, so I've gone ahead and removed all of them in 26c591a9664c8dcd0f3755e685f349b7f9bd488e.